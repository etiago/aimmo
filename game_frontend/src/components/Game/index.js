import styled from 'styled-components'

const Game = styled.section`
  background-color: purple
`

export default Game
