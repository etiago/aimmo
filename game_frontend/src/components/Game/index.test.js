/* eslint-env jest */
import React from 'react'
import Game from 'components/Game'
import renderer from 'react-test-renderer'

describe('<Game />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Game />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
