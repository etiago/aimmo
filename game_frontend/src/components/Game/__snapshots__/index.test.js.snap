// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> renders correctly 1`] = `
Array [
  <nav
    className="sc-bdVaJa dHptmf"
  />,
  <div
    className="sc-bwzfXH eDaynt"
  />,
]
`;
